stable/ghost/templates/NOTES.txt

```diff
@@ -31,9 +32,9 @@ host. To configure Ghost with the URL of your service:
 {{- if eq .Values.serviceType "ClusterIP" }}
 
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app={{ template "fullname" . }}" -o jsonpath="{.items[0].metadata.name}")
-  echo Blog URL  :  http://127.0.0.1:2368/
-  echo Admin URL : http://127.0.0.1:2368/ghost
-  kubectl port-forward $POD_NAME 2368:2368
+  echo Blog URL  :  http://127.0.0.1{{ if .Values.ghostPort }}:{{ .Values.ghostPort  }}{{ end }}/
```

Comment:
I wouldn't use `kubectl port-forward` for this, which talks to one pod only
Instead I'd use the service that does all this for us:
```
export SVC_IP=$(kubectl get service --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.spec.clusterIP}")
echo Blog URL:  http://${SVC_IP}{{ if .Values.ghostPort }}:{{ .Values.ghostPort  }}{{ end }}
echo Admin URL: http://${SVC_IP}{{ if .Values.ghostPort }}:{{ .Values.ghostPort  }}{{ end }}/ghost
```